docs/administration.md

```diff
@@ -253,6 +253,65 @@ You can export your contents of your whole FAQ or just some selected categories
 - a plain HTML5 file
 - a PDF file with a table of contents
 
+### 5.4.3 Batch upload via phpMyAdmin
+<!-- Can I create a multi-line NOTE without spaces between lines? -->
+> [!NOTE]
+> You should have a table ready with the data you want to import. The titles of your new FAQ's in one column, the HTML-formatted content in the next and the keywords in a third column.
+> 
+> You should have access to phpMyAdmin via your web host.
+> 
+> You should have already created a few entries manually in your phpMyFAQ.
+> 
+> This guide is intentionally extensive, but the individual steps can be completed quite quickly once you understand them. It is important that you don't make any mistakes when uploading to your DB, so take your time the first time you try it.
+
+### Export the necessary tables
+
+Open phpMyAdmin. First check whether the correct database is displayed. You can switch to a different database at the top left. At the top left you can see the name of your database and below that the names of the individual tables. From this list, select the following 4 tables (one after the other) and click on them once to select them:
+* faqdata
+* faqdata_tags
+* faqcategoryrelations
+* faqdata_user.csv
+
+![Screenshot of choosing DB and tables in phpmyadmin.](https://pokerfit.de/forum/phpmyadmin-sc-1.png)
```

Comment:
The images should be moved to this repo

Reply:
How can I move the images to this repo? I searched a while, but didn't find it :(